README.md

```diff
@@ -37,6 +38,8 @@ When this is installed you should be able to browse to:
 
 This is for people who want to get a local version up and running quickly to contribute to the codebase of the project.
 
+Set your Neo4j uri and password in [easy_install.sh](https://github.com/pontikos/phenopolis/blob/master/easy_install.sh).
```

Comment:
wrong link - https://github.com/phenopolis/phenopolis/blob/master/easy_install.sh

Reply:
Well spotted. Changed it, thanks.